taxi/tests/test_views.py

```diff
@@ -0,0 +1,167 @@
+
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.forms import ManufacturerSearchForm, CarSearchForm, DriverSearchForm
+from taxi.models import Driver, Car, Manufacturer
+
+
+class PublicViewTests(TestCase):
+    def setUp(self):
+        self.index_url = reverse("taxi:index")
+        self.manufacturer_list_url = reverse("taxi:manufacturer-list")
+        self.manufacturer_create_url = reverse("taxi:manufacturer-create")
+        self.car_list_url = reverse("taxi:car-list")
+        self.driver_list_url = reverse("taxi:driver-list")
+        self.driver_create_url = reverse("taxi:driver-create")
+
+    def test_login_required(self):
+        urls = [self.index_url,
+                self.manufacturer_list_url,
+                self.manufacturer_create_url,
+                self.car_list_url,
+                self.driver_list_url,
+                self.driver_create_url]
+
+        for url in urls:
+            response = self.client.get(url)
+            self.assertNotEquals(response.status_code, 200)
+
+
+class IndexViewTests(TestCase):
+    def setUp(self):
+        self.driver = Driver.objects.create_user(
+            username="test_driver",
+            password="PASSWORD",
+            license_number="ASD1221"
+        )
+        self.index_url = reverse("taxi:index")
+        self.user = Driver.objects.create_user(
+            username="test",
+            password="password")
```

Comment:
In case there are no arguments on the same line with an opening parenthesis, your closing parenthesis should not have any arguments on the same line as well. Please, resolve this issue